# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/linux/reporter/SandboxReporterWrappers.cpp
# Commit: 10e01cf10faf
# Full Hash: 10e01cf10fafca4a150c2f76b3ce4e33bbd06223
# Author: Michael Froman <mfroman@mozilla.com>
# Date: 2020-03-09 21:52:54
# Regressor Bug: 1608558
# File Overlap Count: 1
# Description:
#   Bug 1608558 - pt1 - add linux sandboxing to socket process. r=gcp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60014
# ==============================================================================

diff -r 61e14505a488 -r 10e01cf10faf security/sandbox/linux/reporter/SandboxReporterWrappers.cpp
--- a/security/sandbox/linux/reporter/SandboxReporterWrappers.cpp	Fri Mar 06 20:29:47 2020 +0000
+++ b/security/sandbox/linux/reporter/SandboxReporterWrappers.cpp	Mon Mar 09 14:56:03 2020 +0000
@@ -79,6 +79,9 @@
     case SandboxReport::ProcType::RDD:
       aProcType.AssignLiteral("dataDecoder");
       return NS_OK;
+    case SandboxReport::ProcType::SOCKET_PROCESS:
+      aProcType.AssignLiteral("socketProcess");
+      return NS_OK;
     default:
       MOZ_ASSERT(false);
       return NS_ERROR_UNEXPECTED;
